content/renderer/media/media_factory.cc

```diff
@@ -752,7 +763,11 @@ MediaFactory::CreateRendererFactorySelector(
     // TODO(b/326827007): Revisit renderer to support secondary videos.
     factory_selector->AddFactory(
         RendererType::kStarboard,
-        std::make_unique<media::StarboardRendererFactory>(media_log));
+        std::make_unique<media::StarboardRendererFactory>(
+            media_log,
+            // TODO: Inject these values
```

Comment:
Will create a bug for this once this PR is approved to merge.

Reply:
Please use the TODO format here: cl/702469138